requirements-dev.txt

```
@@ -9,3 +9,4 @@ pytest-benchmark
 black>=21
 tomli~=2.0.0 # Drop once minimum Python version is 3.11
 isort==5.13.2
+pylint==2.7.4
```

```suggestion
pylint==2.7.4

```
Very minor: https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log